pkgs/applications/science/robotics/betaflight-configurator/default.nix

```diff
@@ -0,0 +1,52 @@
+{stdenv, fetchurl, unzip, makeDesktopItem, nwjs }:
+
+stdenv.mkDerivation rec {
+  strippedName = "betaflight-configurator";
```

Comment:
Could this be moved from the derivation into a `let` binding?  Same with `desktopItem` perhaps?

Reply:
Yes! I did accordingly

Reply:
Sorry for not seeing this sooner.  It all looks good to me.